### PR TITLE
[MODULAR][HOST-REQUEST] Minimum Length Flavor Text Requirement

### PR DESCRIPTION
**Pull request by ZephyrTFA**
## Host Request

Simply put; if you do not have a flavor text, or it is shorter than the cut off set by staff, you cannot ready up or join an active round.

**Verified Host Input - See Comment Section by Host:** It's a 150 character limits without whitespace:

**Example:** Here is a human who is rather plain in appearance with short, brown hair and freckles. Looking well into his thirties, he appears constantly tired if not exhausted. Some parts of his body part appear rather slimy

That's 178 characters without a whitespace. It's really not that much. I wrote that in half a minute.

Anyways, this will be merged in some form once we announce our character/species standard guideline which has been in discussion for over a month now in public.

### CODE REVIEW

**Comment by hugmeorelse** [comment]
I edited the PR description to place my input.

**Comment by Sigmar-alkahest** [comment]
i like this. it makes people have to put atleast the barest fucking effort into describing what their character looks like.

File: modular_skyrat/modules/customization/modules/client/preferences.dm [hunk 1]
```diff
@@ -270,6 +270,24 @@ GLOBAL_LIST_INIT(food, list(
...
 #define APPEARANCE_CATEGORY_COLUMN "<td valign='top' width='14%'>"
 #define MAX_MUTANT_ROWS 4
+#define FLAVORTEXT_JOIN_MINIMUM 150
+
+/datum/preferences/proc/check_flavor_text(inform_client = TRUE)
+	if(!features["flavor_text"])
+		if(inform_client)
+			to_chat(parent, "<span class='userdanger'>You must have a flavor text!</span>")
+		if(check_rights(R_ADMIN))
+			message_admins("[usr] is bypassing the flavor text requirements.")
```
Comment (FlamingLily) [inline modular_skyrat/modules/customization/modules/client/preferences.dm]: Does this throw an alert to admins saying "Do you wish to bypass flavour text requirements?" or not, cause if not, it probably should

**Comment by Stalkeros2** [comment]
This says a lot about our society.

File: modular_skyrat/modules/customization/modules/client/preferences.dm (hunk 1 above)
Comment (hugmeorelse) [reply modular_skyrat/modules/customization/modules/client/preferences.dm]: > Does this throw an alert to admins saying "Do you wish to bypass flavour text requirements?" or not, cause if not, it probably should

Yes, we typically want to surpass it when it comes for using quick characters for events, and some admins may forget about the flavor text limit.

**Comment by NullDagaf** [comment]
Not a big fan of this personally, would much rather see a deepened character customization over just forcing people to summarise how their character's sprite looks. Perhaps a height/body type setting could be ported over, which would be shown in examine (as shown in the image linked), or bring back the scar customization (unsure if that's already implemented or not).
better_exam

Also won't lie, it's quite hilarious to think about potential "flavor text bypassers" that'd just link lorem ipsum in the flavor text.

**Pull request merged**